packages/preview-middleware-client/test/unit/adp/controllers/AddFragment.controller.test.ts

```diff
@@ -733,5 +733,222 @@ describe('AddFragment', () => {
                 templateName: 'OBJECT_PAGE_CUSTOM_SECTION'
             });
         });
+
+        test('add header field fragment and a change case 1', async () => {
```

Comment:
Could you please update the names of the tests with _exactly the thing that they are doing_? Because **case 1** is not descriptive enough. Imagine the order of the **if statements** changes or you keep adding stuff there and the tests become unmaintainable.

Also the ones below.

Reply:
done.